text/0000-slatepack.md

```diff
@@ -0,0 +1,384 @@
+
+#### Formatting
+
+- `WORD_LENGTH`: `15`
+  - Number of `SimpleBase58Check` encoded characters per word; chosen for human readability across device screen sizes
+
+- `LINE_BREAK`: `200` words
+  - Number of words of `WORD_LENGTH` to include before inserting a newline; chosen for user friendliness in terminals and messaging applications
+
+- `MESSAGE_BREAK`: `X` lines
+  - Number of lines an armored message formatted with `WORD_LENGTH` and `LINE_BREAK` can contain before it must be split into multiple message parts
+  - This parameter still needs to be selected, ideally to cover the most possible likely slate sizes
+
+- `MAX_MESSAGES`: `9`
+  - The maximum number of messages that can be used to represent a slate payload
+
+All chosen parameters except for `MAX_MESSAGES` should be changeable without breaking armored slates as whitespace and newlines are removed during decoding and in the case of message breaks, the error check code is over the entire slate payload. `MAX_MESSAGES` cannot exceed 9 because they would be rejected by clients as invalid since the regex will not match. Slates larger than this should use synchronous transport for a better user and developer experience.
+
+### Example SlatePack JSON Object
+
+#### Plain Text
+
+In this plain text example, neither the sender nor the receiver wish to share a `SlatePack Address`
+
+```
+{
+  "slatepack": [0, 1],
+  "mode": 0,
+  "sender": "0",
+  "payload": <binary serialized slate>
+}
+```
+
+#### Encrypted
+
+```
+{
+  "slatepack": [0, 1],
+  "mode": 1,
+  "sender": "grin1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x",
+  "header": {
+    "recipients_list": {
+        "0": {
+            "epk": "SVrzdFfkPxf0LPHOUGB1gNb9E5Vr8EUDa9kxk04iQ0o",
+            "emk": "0OrTkKHpE7klNLd0k+9Uam5hkQkzMxaqKcIPRIO1sNE"
+        }
+    },
+    "mac": "gxhoSa5BciRDt8lOpYNcx4EYtKpS0CJ06F3ZwN82VaM"
+  },
+  "payload": <age encrypted binary seriaized slate binary>,
+}
+```
+
+### Example `SlatePack Message`
+
+```
+BEGINSLATEPACK. 4H1qx1wHe668tFW yC2gfL8PPd8kSgv
+pcXQhyRkHbyKHZg GN75o7uWoT3dkib R2tj1fFGN2FoRLY
+GWmtgsneoXf7N4D uVWuyZSamPhfF1u AHRaYWvhF7jQvKx
+wNJAc7qmVm9JVcm NJLEw4k5BU7jY6S eb. ENDSLATEPACK.
+```
+
+## `SlatePack Workflow`
+
+Adoption of the SlatePack standard allows for a unified workflow that can still function without knowledge of a `SlatePack Address` from a counterparty.
+
+### With a `SlatePack Address`
+
+1. `grin-wallet send -d grin1bech32address 1.337`
+
+2. Sender wallet derives an onion v3 address from `grin1bech32address` and attempts to complete the transaction synchronously via Tor
+
+3. (Fallback) If the synchronous transaction fails, a `SlatePack Message` string is encrypted to the `SlatePack Address` and output for manual asynchronous transport by the user
+
+### Without a `SlatePack Address`
+
+1. `grin-wallet send -d slatepack 1.337`
```

Comment:
why not simply `grin-wallet send 1.337`?

Reply:
Great simplification opportunity :rocket: